src/main/telemetry/frsky.c

```diff
@@ -190,7 +191,14 @@ static void sendThrottleOrBatterySizeAsRpm(void)
 {
     sendDataHead(ID_RPM);
     if (ARMING_FLAG(ARMED)) {
-        serialize16(rcCommand[THROTTLE] / BLADE_NUMBER_DIVIDER);
+        if(rcCommand[THROTTLE] <= escAndServoConfig->minthrottle){
```

Comment:
Feeling pretty stupid ATM!!
I've tried rcData[THROTTLE] < rxConfig->mincheck, but rcData[THROTTLE] is returning values from 990 to 2015 (fairly expected), but mincheck is returning 2045, so the comparison never evaluates to true (no matter how much throttle is applied).  Not sure where this mincheck value is coming from.